doc/nrf/ug_bt_mesh_configuring.rst

```diff
@@ -34,3 +34,10 @@ Mesh models
 
 The |NCS| Bluetooth mesh model implementations are optional features, and each model has individual Kconfig options that must be explicitly enabled.
 See :ref:`bt_mesh_models` for details.
+
+Additional configuration options
+================================
+
+This section lists additional configuration options that can be used to configure behavior and performance of Bluetooth mesh. This section will be expanded in future to include relavent configuration options.
```

Comment:
Is the last sentence necessary? At what point do we remove it, so I would like to remove it.